util/build_configs/cray-internal/setenv-xc-x86_64.bash

```diff
@@ -334,10 +334,9 @@ else
         list_loaded_modules
     fi
 
-    # Please keep the gen versions in compiler_versions.bash the same as these!
-    gen_version_gcc=7.3.0
+    gen_version_gcc=9.3.0
```

Comment:
I don't think we want to change our x86 gen compiler. This is a configuration where we care about the gen compiler being compatible with a wide variety of potential target compilers.